content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -1,56 +1,190 @@
+2. Click the Roles tab and click on Create Role in the IAM sidebar.
 
-## One-click cluster creation with Restack
+ ![](https://cdn.sanity.io/images/ev3amoz3/production/7d1ba16ba5610e8947ca20d8746ee8424a9d4621-2000x1062.png)
+
+3. In Select type of trusted entity, click the Custom trust policy
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/666cb5015b3c5a29e6eef6216bdd97ef67297dc8-1770x1110.png)
+
+4. Copy and Paste into the custom policy field the following. You also can copy it directly from the Restack user interface modal.
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::223746943638:user/connect",
+                    "arn:aws:iam::223746943638:role/connect"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "restack"
+                }
+            }
+        }
+    ]
+  }
+```
+
+5. Click the Next button and Add AdministratorAccess Permissions.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/2685ee99b153f46b38f99607407dfba72c5c6522-2000x1160.png)
+
+6. Click the Next button and Add Role Name `restack`, Review, and Create button.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/fb14345d290077195b986cb3931c0d39689dca6f-2000x1255.png)
+
+7. From IAM > Role overview, navigate to the role you just created and copy the ARN from there.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/e3750c4fb52873a781e328e81c70237d1968d5da-2560x1440.png)
+
+8. Copy the ARN to Restack's user interface and check by clicking on 'Connect cloud' if Restack can successfully assume the role in your AWS account.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/ef820e88fd8cd5bd64ca24b79f22bc96cf9fd255-2560x1440.png)
+
+If the cloud account was added successfully and Restack can connect to it, you can proceed to create a cluster and provision applications.
+
+
+`sts:AssumeRole` is an AWS Security Token Service (STS) API operation that allows an AWS Identity and Access Management (IAM) user or role in one account to assume a role in another AWS account. This allows the user or role to access resources in the second account that they would not normally have access to.
+
+The `sts:AssumeRole` operation returns a set of temporary security credentials that the user or role can use to access resources in the second account. These credentials include an access key, a secret access key, and a session token, which can be used to make AWS API requests.
+
+
+## Connect your GCP account
+
+Connect your GCP account in a few steps to Restack.
+
+> Requirements
+- Active Google Cloud Organization
+- Active Google project for all Restack related resources
+- Active billing profile associated with the project
```

Comment:
```suggestion
First, make sure that the following requirements are satisfied:

1. You have an active Google Cloud account or organization
2. You have an active project that can be used to deploy resources from Restack
3. You have an active billing profile associated with that project
```

I'm assuming that both a Google Cloud account AND an organization will work? Or does it only work if you have an organization on GCP? It is worth cross-checking as we have users who would like to deploy Kestra with Restack to their private GCP account without having an organization on GCP